src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,24 @@
+package duke.exception;
+
+/**
+ * Represents a checked exception thrown when the chatbot
+ * encountered an error state.
+ */
+public class DukeException extends Exception {
```

Comment:
Love that you have Duke exception with multiple classes to handle the different exceptions